### DISCUSSION TITLE
CSS inheritance between CMS and my site

### DISCUSSION THREAD

**Discussion by scottgallant** [tinacms/tinacms#2944] [General]
I set up TinaCMS on a couple of sites recently and found there to be a few places where unwanted CSS was shared between the CMS and my site.   See example: 

<img width="847" alt="Screen Shot 2022-06-01 at 2 49 03 PM" src="https://user-images.githubusercontent.com/776019/171470719-be0bf53b-090b-47fc-98af-b72f0d1dd602.png">

Have others found that?  Is there any way to put up a wall between these two code bases? 

Edit: the arrows in that screenshot aren't exactly accurate 😅. I meant to point to the headings with the serif font (applied to my site).

**Comment by spbyrne** [comment]
I'll fix these specific cases, post pictures of anything like this and I'll try to patch it. We've talked about the possibility of exploring using iframes again, which would solve this but add different problems to overcome. There is a global font reset on the `.tina-tailwind` class which catches a lot of this, but if I get too aggressive there it becomes impossible to use tailwind on components within Tina.

**Reply by jeffsee55** [reply]
Is there a downside to setting some of the more obvious ones with `tina-tailwind *`?

```css
.tina-tailwind * {
  font-family: theme('fontFamily.sans'),
  font-color: etc etc 
}
```
